admin_guide/out_of_resource_handling.adoc

```diff
@@ -99,51 +240,101 @@ deprecated in favor of a pure disk eviction based configuration.
-=== Eviction Thresholds
+=== Understanding Eviction Thresholds
 
 You can configure a node to specify eviction thresholds, which trigger the node
-to reclaim resources.
+to reclaim resources, by adding a threshold to the xref:out-of-resource-eviction-policy[node configuration file].
 
-Eviction thresholds can be soft, for when you allow a grace period before
-reclaiming resources, and hard, for when the node takes immediate action when a
-threshold is met.
+If an eviction threshold is met, independent of its associated grace period, the
+node reports a condition indicating that the node is under memory or disk pressure. The node continues to report node status updates at the
+frequency specified by the `node-status-update-frequency` argument, which
+defaults to `10s`.
+
+Eviction thresholds can be xref:out-of-resource-hard-eviction-thresholds[hard], for when the node takes immediate action when a
+threshold is met, or xref:out-of-resource-soft-eviction-thresholds[soft], for when you allow a grace period before
+reclaiming resources.
+
+[NOTE]
+====
+Soft eviction usage is more common when you are targeting a certain level of
+utilization, but can tolerate temporary spikes. It is recommended
+that the soft eviction threshold is always less than the hard eviction
+threshold, but the time period is operator-specific. The system reservation
+should also cover the soft eviction threshold.
+
+The soft eviction threshold is an advanced feature. You should configure a hard eviction threshold before attempting to use soft eviction thresholds. 
+====
 
 Thresholds are configured in the following form:
 
 ----
 <eviction_signal><operator><quantity>
 ----
 
-* Valid `eviction-signal` tokens as defined by eviction signals.
-* Valid `operator` tokens are `<`.
-* Valid `quantity` tokens must match the link:https://github.com/kubernetes/kubernetes/blob/master/docs/design/resources.md#resource-quantities[quantity representation] used by
-Kubernetes.
-* an eviction threshold can be expressed as a percentage if it ends with the `%` token.
+* the `eviction-signal` value can be any xref:out-of-resource-eviction-signals-supported[supported eviction signal].
+* the `operator` value is `<`.
+* the `quantity` value must match the link:https://github.com/kubernetes/community/blob/master/contributors/design-proposals/resources.md#resource-quantities[quantity representation] used by
+Kubernetes and can be expressed as a percentage if it ends with the `%` token.
 
 For example, if an operator has a node with 10Gi of memory, and that operator
 wants to induce eviction if available memory falls below 1Gi, an eviction
 threshold for memory can be specified as either of the following:
 
----
+----
 memory.available<1Gi
 memory.available<10%
----
+----
+
+[[out-of-resource-eviction-monitoring-interval]]
+
+[NOTE]
+====
+The node evaluates and monitors eviction thresholds every 10 seconds and the
+value can not be modified. This is the housekeeping interval.
```

Comment:
Added: "The cAdvisor housekeeping interval is 30 seconds by default."
The node auto-discovers these file systems using `cAdvisor`. The cAdvisor housekeeping interval is 30 seconds by default.